src/calendar/day/period/index.tsx

```diff
@@ -130,8 +132,8 @@ export default class PeriodDay extends Component<PeriodDayProps>{
     const {state, marking} = this.props;
     const containerStyle = [this.style.base];
     const textStyle = [this.style.text];
-    let leftFillerStyle = {backgroundColor: undefined};
-    let rightFillerStyle = {backgroundColor: undefined};
+    let leftFillerStyle: {backgroundColor?: string} = {backgroundColor: undefined};
+    let rightFillerStyle: {backgroundColor?: string} = {backgroundColor: undefined};
```

Comment:
This type is going to limit the style to have only 'backgroundColor' key... I think this is a breaking change. Did you test it?

Reply:
I did test it.. anyway - I changed it to ViewStyle